docs/concepts/event-handler.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Event Handler Concept
+headline: Event Handler
+bodyclass: docs
+layout: docs
+sidenav: doc-side-concepts-nav.html
+type: markdown
+---
+<h2 class="top">Event Handler</h2> 
```

Comment:
Let's rename it “Event Subscriber”. That way we'd differentiate ”_handling_ commands” with ”_subscribing_ to events”.